src/components/Table/options/expanded/hook.ts

```diff
@@ -0,0 +1,6 @@
+import { ExpandedOptions, RowData } from "@tanstack/react-table";
+import { EXPANDED_OPTIONS } from "./constants";
+
+export function useExpandedOptions<T extends RowData>(): ExpandedOptions<T> {
+  return { ...EXPANDED_OPTIONS };
```

Comment:
Scaffolding structure, set up and ready for when we want to implement controlling expansion of rows.